gui.md

```
@@ -4,100 +4,162 @@
+
+#### Fields
+
+* `name`
+  * **Required**: Yes
+  * **Description**: The human-readable label displayed for this input field in the GUI. Can include template variables like `${order.outputs.0.token.symbol}`.
+* `description`
+  * **Required**: No
+  * **Description**: An optional, longer description or help text displayed for this field, potentially as a tooltip or helper text.
+* `presets`
+  * **Required**: No
+  * **Description**: An optional list of [Preset Items](#preset-item) containing predefined values the user can select for this field.
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid

(MD051, link-fragments)